js/samples/04.ai-apps/e.assistants-orderBot/README.md

```diff
@@ -75,43 +70,42 @@ To use Teams Toolkit, continue following the directions below.
 
-<!-- COMMENTED OUT UNTIL AZURE OPENAI IS AVAILABLE FOR ASSISTANTS
-If you are using Azure OpenAI then follow these steps:
--   Comment the `SECRET_OPENAI_KEY` variable in the `./env/.env.local.user` file.
--   Add your Azure OpenAI key and endpoint values to the `SECRET_AZURE_OPENAI_KEY` and `SECRET_AZURE_OPENAI_ENDPOINT` variables
--   Open the `teamsapp.local.yml` file and modify the last step to use Azure OpenAI variables instead:
 ```yml
 - uses: file/createOrUpdateEnvironmentFile
     with:
       target: ./.env
       envs:
         BOT_ID: ${{BOT_ID}}
         BOT_PASSWORD: ${{SECRET_BOT_PASSWORD}}
-        #OPENAI_KEY: ${{SECRET_OPENAI_KEY}}
-        AZURE_OPENAI_KEY: ${{SECRET_AZURE_OPENAI_KEY}}
-        AZURE_OPENAI_ENDPOINT: ${{SECRET_AZURE_OPENAI_ENDPOINT}}
+        OPENAI_KEY: ${{SECRET_OPENAI_KEY}}
+        # AZURE_OPENAI_KEY: ${{SECRET_AZURE_OPENAI_KEY}}
+        # AZURE_OPENAI_ENDPOINT: ${{SECRET_AZURE_OPENAI_ENDPOINT}}
 ```
--   Open `./infra/azure.bicep` and comment out lines 72-75 and uncomment lines 76-83.
--   Open `./infra/azure.parameters.json` and replace lines 20-22 with:
+
+-   Open `./infra/azure.bicep` and comment out lines 69-76 and uncomment lines 77-80.
+-   Open `./infra/azure.parameters.json` and replace lines 20-25 with:
+
 ```json
-      "azureOpenAIKey": {
-        "value": "${{SECRET_AZURE_OPENAI_KEY}}"
-      },
-      "azureOpenAIEndpoint": {
-        "value": "${{SECRET_AZURE_OPENAI_ENDPOINT}}"
-      }
+"openAIKey": {
+    "value": "${{SECRET_OPENAI_KEY}}"
+},
 ```
-<-->
 
 1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
-1. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
-1. Copy this sample into a new folder outside of teams-ai
-1. Select File > Open Folder in VS Code and choose this sample's directory
-1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
-1. Ensure that you have set up the sample from the previous step.
-1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
-1. In the browser that launches, select the **Add** button to install the app to Teams.
+2. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+3. Copy this sample into a new folder outside of teams-ai
+4. Select File > Open Folder in VS Code and choose this sample's directory
+5. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
+6. Ensure that you have set up the sample from the previous step.
+7. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+8. In the browser that launches, select the **Add** button to install the app to Teams.
```

Comment:
You don't need to manually number this. If they are all 1, markdown will bump the numbers sequentially automatically. This is helpful when we need to add or remove steps later and don't need to manually update each number.

Reply:
my vscode auto did that lol - i'll revert it